tests/Directory.Build.targets

```diff
@@ -20,6 +20,9 @@
 
   <ItemGroup>
     <None Include="$(XunitRunnerJson)" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(MSBuildProjectDirectory)\Snapshots\**\*"
```

Comment:
Does this copy the files to the bin directory even when I'm building for the local build? We never use them there, right? We would always use them in the repo / project structure. Can we not copy them when we aren't using Helix?